docs/user-manual/booking-system.md

```diff
@@ -0,0 +1,144 @@
+The `BookStore` is an Object which managed Histograms for later universal access.
```

Comment:
Can you move this file to a new directory `internal-documentation` instead of `user-manual` ?